imageformats/oci/importer.go

```diff
@@ -1,4 +1,5 @@
-package containerd
+// Package oci provides the importer and the exporter for OCI Image Spec.
```

Comment:
Did we not already remove this package?

Why is this under a new package heirarchy, `imageformats`? Don't we already have a package heirarchy called `images`?

Please use that. I set it up for a reason.

Reply:
This OCI pkg is different from the removed pkg with the same name, but I can move this to under images.

I'll update pr